cmd/podman/common/util.go

```diff
@@ -200,6 +200,10 @@ func parseSplitPort(hostIP, hostPort *string, ctrPort string, protocol *string)
 			}
 			newPort.HostPort = hostStart
 		}
+	} else {
+		// Assume a direct host to container mapping if the provided host port
+		// is nil.
```

Comment:
Is this documented on a man page(s) somewhere?

Reply:
I think this does not reflect the new implementation target. I'll remove this code part.